sst/config/config.py

```diff
@@ -0,0 +1,61 @@
+import sst
+
+DEBUG_L1 = 0
+DEBUG_MEM = 0
+DEBUG_LEVEL = 0
+
+# Define the simulation components
+cpu = sst.Component("core", "sstsimeng.simengcore")
+cpu.addParams({
+    "config_path": "/home/rahat/asimov/SimEng/configs/a64fx.yaml",
+    "executable_path": "/home/rahat/asimov/simeng-progs/stream",
+    "executable_args": "",
+    "clock" : "1GHz",
+    "max_addr_memory": 1*1024*1024*1024-1,
```

Comment:
Shouldn't this be implicit from the size of HEAP and STACK defined in the model config?